setup.md

```diff
@@ -0,0 +1,156 @@
+# OrcaDetector setup
+
+This UC Berkeley Master of Information in Data Science capstone project was developed by
+[Spyros Garyfallos](https://github.com/paloukari), [Ram Iyer](https://github.com/ram-iyer), and [Mike Winton](https://github.com/mwinton).
+
+## 0. Provision a cloud GPU machine
+
+### Using AWS
+
+If using AWS, as assumed by these setup instructions, provision an Ubuntu 18.04 `p2.xlarge` instance.  It's got older GPUs (Tesla K80) but is much cheaper.  Make sure to upgrade the storage space (e.g. 500 GB).  Also, make sure to pick a prebuilt Deep Learning AMI during the first step of the provisioning process. The most current version as of writing is `Deep Learning AMI (Ubuntu) Version 23.0 - ami-058f26d848e91a4e8`. This will already have `docker` and `nvidia-docker` pre-installed and will save you a lot of manual installation "fun".
+
+## 1. Set up required environment variables - UPDATE
+
+Set the environment variable for MLFlow in your system's `~/.bashrc` file (or on a Mac, in your `~/.bash_profile` file):
+
+```
+TBD
+```
+
+After saving the file, type `source ~/.bashrc` to load the new variables into your environment.
+
+
+## 2. Clone the project repo
+
+If you haven't already, clone the project Git repo to your instance.  Doing so in your home directory is convenient, and this document assumes you have done so.
+
+```
+cd ~
+git clone https://github.com/paloukari/OrcaDetector
+```
+
+## 3. Download data files and model weights
+
+First, if you don't already have the AWS CLI installed, follow the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html).
+
+### Marine mammal audio samples
+
+We have downloaded the audio files from the [Watkins Marine Mammal Sound Database](https://cis.whoi.edu/science/B/whalesounds/fullCuts.cfm) using the [/orca_detector/getData.py](./orca_detector/getData.py) script, based on a sample provided to us by Watkins.
+
+Expand `data.tar.gz` inside the main directory of the repo; it will create a `/data` folder.  (There are `.gitignore` rules in place to prevent it from accidentally being submitted to GitHub.)
+
+```
+cd ~/OrcaDetector
+aws s3 cp s3://w251-orca-detector-data/data.tar.gz .
+tar -xvf data.tar.gz
+```
+
+Alternately you can download [data.zip](https://drive.google.com/file/d/10mGIptby8SEf4yk0m57mVtiXCgRQrgTc/view?usp=sharing) from Google Drive, but due to the size of the archive, it's probably much faster to pull it from the S3 bucket.
```

Comment:
!! I'm not sure we're allowed to redistribute the wav files, I'd take this out

Reply:
I'll leave the links in for now since this guide for now is just targeting us, but I agree we should remove these links (and shut down public access to the bucket) when we're done.  Issue filed.